Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM node:lts-alpine as app
 ARG NODE_ENV=production
+ARG GCAL_CLIENT_SECRET
```

Comment:
I think this needs to be prefixed with `REACT_APP` instead of `GCAL` so it matches your `deploy.yml` file.

Reply:
Wow thanks for catching that! I forgot I edited this doc. Fixed!